crates/tikv-rust-client-wasmtime/src/lib.rs

```diff
@@ -0,0 +1,74 @@
+//! Impement the WASI cache interface using a Redis instance.
+//! This is using a Wasmtime host implementation.
+
+use std::sync::Arc;
+use tokio::runtime::Runtime;
+use lazy_static::lazy_static;
+
+use wasi_cache::*;
+
+pub use wasi_cache::add_to_linker;
+
+wit_bindgen_wasmtime::export!("wit/ephemeral/wasi-cache.wit");
+lazy_static! {
+    pub(crate) static ref RUNTIME: Runtime = Runtime::new().unwrap();
```

Comment:
Do we need a global static Tokio runtime?
Could the `TikvClient` functions be async, and `block_on` them in the cache impl?

Reply:
Do you actually need a Tokio runtime?
Could you use `futures::block_on` instead of a static runtime?

Reply:
yes because tikv client uses tokio as internal async driver